[Sprig App] Zombie Rush

arghya-v opened the pull request:
### Author name
Author: arghya-v

### About your game

**What is your game about?**
A cool little game I made using the Hackclub Sprig engine, you kill zombies to get to the next level!

**How do you play your game?**
W A S D to move

J to shoot left
L to shoot right

whatwareweb commented:
ok metadata looks good to me, a reviewer will get to your PR shortly

whatwareweb commented:
Hi! Your game looks good, and the music is nice. Could you add some additional levels to your game to increase the time? Currently it's quite short.

In addition, there is no way to lose, so the game is quite easy. Could you add some elements to make the game more challenging? Thanks!

whatwareweb reviewed (changes requested):
requesting changes as described above

arghya-v commented:
hey! I just updated the game, I added the ability for zombies to get closer to you. If you touch a zombie, you lose. I also added 2 more levels,

arghya-v commented:
> requesting changes as described above

Hey! I just updated the repo. I made it so that if you touch a zombie, you lose. I added the ability for the zombies to get closer to you as you move, to make it challenging. And I added 2 more levels.

whatwareweb reviewed (changes requested):
Just played it, love this update!! Two things before we merge, can you make a way to reset the game? This makes it easier for people on the real console.

Also, can you fix these errors i get while playing?
<img width="601" height="151" alt="image" src="https://github.com/user-attachments/assets/79eb6b10-c872-457d-b477-a418ed595977" />

arghya-v commented:
> Just played it, love this update!! Two things before we merge, can you make a way to reset the game? This makes it easier for people on the real console.
> 
> Also, can you fix these errors i get while playing? <img alt="image" width="601" height="151" src="https://private-user-images.githubusercontent.com/59752545/498463017-79eb6b10-c872-457d-b477-a418ed595977.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTk4NzUzMjQsIm5iZiI6MTc1OTg3NTAyNCwicGF0aCI6Ii81OTc1MjU0NS80OTg0NjMwMTctNzllYjZiMTAtYzg3Mi00NTdkLWI0NzctYTQxOGVkNTk1OTc3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTEwMDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMDA3VDIyMTAyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRjN2RkOTcxZGI2NTkwNTc1NzM4YTljYWRkOGJhNzg3YjBmNjVjZTFjOTQ1MzZmY2Q0YWI5ZTY2YTYxYzkwZDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.EDaBdgM6TdKAsWx3tlCsoOmWlwMYonv1vfR3dn-kvB4">

just added those! good to merge now
